code/game/turfs/turf.dm

```diff
@@ -320,3 +332,21 @@
 	SSair.remove_from_active(T0)
 	T0.CalculateAdjacentTurfs()
 	SSair.add_to_active(T0,1)
+
+
+/turf/proc/update_cliff()
+	if(cliff_images) //Not cut_overlays() because turf smoothing gets royally fugged.
+		var/list/L = list()
+		for(var/k in cliff_images)
```

Comment:
@PJB3005 that's extra relevant, as I said in the bus (but failed to crosspost here) it was actually C# (.NET) Dictionaries that made me feel sad about this loop (and thus make the Feature Request, which lummox appears to support!)